section2-week1/exercise7/Main.cpp

```diff
@@ -11,93 +11,96 @@ using namespace std;
 
 int main()
 {
-	int value, random, answer;
-	ans = 0;
+	int value, random;
 
-	srand(time(NULL));
-	random = rand() % (1 - 4);
 
 	cout << "Hey you, let's play Rock, Paper, Scissors!" << endl;
-
 	cout << "1 - Rock\n2 - Paper\n3 - Scissors" << endl;
 	cout << "choose you weapon: " << endl;
 
-	cin >> value;
+	enum Weapon
+	{
+		Rock, Paper, Scissors
+	};
+	srand(time(NULL));
+	random = rand() % (1 - 4);
```

Comment:
Random could have a more specific name. What _is_ random? If you read this variable name without any context, would you know what it is? You could specify that it's a weapon, for example. Something like `randomWeapon` or go even further and say `computerWeaponChoice` or `randomWeaponChoice`, something like that. What do you think?

Reply:
Never mind, I see that this is just a random number that you're using to cast to a Weapon. However, `weapon` could also be more descriptive. Is it the user's weapon? Or the computer's weapon?